kotlin-audio/build.gradle

```diff
@@ -5,17 +5,17 @@ plugins {
     id 'de.mannodermaus.android-junit5'
 }
 
-def versionNumber = '2.0.0'
+def versionNumber = '1.0.0'
 
 group = 'com.github.doublesymmetry'
 version = versionNumber
 
 android {
-    compileSdk 31
+    compileSdk 33
```

Comment:
Bumping these versions create a BC break. Is this absolutely necessary? I'm not crazy about it.